guides/common/modules/proc_adding-a-passphrase-to-ssh-key-used-for-remote-execution.adoc

```diff
@@ -0,0 +1,16 @@
+[id="adding-a-passphrase-to-ssh-key-used-for-remote-execution_{context}"]
+= Adding a passphrase to SSH key used for remote execution
+
+By default, {SmartProxy} uses a non-passphrase protected SSH key to execute remote jobs on hosts.
+You can protect the SSH key with a passphrase by following this procedure.
+
+.Procedure
+* On your {ProjectServer} or {SmartProxyServer}, use `ssh-keygen` to add a passphrase to your SSH key:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# ssh-keygen -p -f ~foreman-proxy/.ssh/id_rsa_foreman_proxy
+----
+
+.Next steps
+* Users now must authenticate with a passphrase when running remote execution jobs on hosts.
```

Comment:
Nit: You use the passphrase to unlock the key and then authenticate with the unlocked key. You don't really authenticate with the passphrase

Reply:
Changed wording.